src/components/tree/TreeInnerNode.svelte

```diff
@@ -21,7 +21,18 @@
 </script>
 
 <div class="tv_node">
-  <span on:click={toggleExpanded}>
+  <span
+    on:click={toggleExpanded}
+    role="button"
+    tabindex="0"
+    on:keydown={(e) => {
+      if (e.key !== 'Enter' && e.key !== ' ') return;
+      e.preventDefault();
+      if (e.target != null) {
+        e.target.click();
+      }
+    }}
+  >
```

Comment:
whats the purpose of this?  it looks like an addition in functionality to allow keyboard interaction with the leftmost panel (signal legend), and not a compatibility migration.